### PR TITLE
ci: Add CentOS setup script.

### LINKED ISSUE #197: ci: Add CentOS setup script
In order to cover https://github.com/kata-containers/ci/issues/13, we need to add an installation script for CentOS

### PR DESCRIPTION
**Pull request by GabyCT**
This will enable to setup and run kata-runtime in CentOS.

Fixes #197

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: .ci/setup_env_centos.sh [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+cidir=$(dirname "$0")
+source "/etc/os-release"
+source "${cidir}/lib.sh"
+
+# Obtain CentOS version
+centos_version=$(cat /etc/os-release | grep "VERSION_ID" | cut -d '"' -f2)
+
+# Enable EPEL repository on CentOS
+sudo -E yum install -y wget rpm
+wget http://dl.fedoraproject.org/pub/epel/epel-release-latest-${centos_version}.noarch.rpm
```
Comment (jodh-intel) [inline .ci/setup_env_centos.sh]: This should be `https://`.

File: .ci/setup_env_centos.sh [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+#
+
+set -e
+
+cidir=$(dirname "$0")
+source "/etc/os-release"
+source "${cidir}/lib.sh"
+
+# Obtain CentOS version
+centos_version=$(cat /etc/os-release | grep "VERSION_ID" | cut -d '"' -f2)
```
Comment (jodh-intel) [inline .ci/setup_env_centos.sh]: Could be simplified to:

```
centos_version=$(grep VERSION_ID /etc/os-release | cut -d '"' -f2)
```

**Comment by GabyCT** [comment]
@chavafg , I think cri-o is failing
Makefile:35: recipe for target 'crio' failed
make: *** [crio] Error 1
Build step 'Execute shell' marked build as failure
Performing Post build task...
Could not match :Build was aborted  : False
Any ideas?

File: .ci/lib.sh [hunk 3]
```diff
@@ -56,7 +56,9 @@ function get_version(){
 	versions_file="$GOPATH/src/github.com/kata-containers/runtime/versions.yaml"
 	go get -d -u -v "$runtime_repo" || true
 	[ ! -f "$versions_file" ] && { echo >&2 "ERROR: cannot find $versions_file"; exit 1; }
-	yq read "$versions_file" "$dependency"
+	if [ "$ID" == "ubuntu" ] || [ "$ID" == "fedora" ]; then
```
Comment (chavafg) [inline .ci/lib.sh]: If you add this condition, centos will not be able to read the `version.yaml` file and we would not be able to get the versions we need.

File: .ci/install_qemu.sh [hunk 4]
```diff
@@ -37,4 +39,9 @@ sudo -E make install
 # in a repository.
 sudo ln -sf $(command -v qemu-system-$(arch)) /usr/bin/
 
+if [ "$ID" == "centos" ]; then
+	qemu_path="/usr/local/bin/qemu-system-$(arch)"
+	sudo ln -sf "$qemu_path" /usr/bin/
```
Comment (chavafg) [inline .ci/install_qemu.sh]: I think this is not needed anymore as we are already creating the symlink above.  See L40.

Comment (GabyCT) [reply .ci/install_qemu.sh]: @chavafg centos does not retrieve a value with $(command -v qemu-system-$(arch))

**Comment by GabyCT** [comment]
@chavafg changes applied

**Comment by chavafg** [comment]
lgtm

**Pull request merged**